### DISCUSSION TITLE
How to customize nixpacks.toml?

### DISCUSSION THREAD

**Discussion by creeppug** [Dokploy/dokploy#418] [Q&A] [answered]
I am deploying a python application and after installing the libraries I need to enter one command(playwright install). Is there any way to customize nixpacks.toml ? I searched for this file on my server, but could not find it.

**Comment by Siumauricio** [comment, marked as answer]
you need to create byyourself the nixpacks.toml in your project and when you deploy it automatically will recognize

**Reply by creeppug** [reply]
Thank you!
